app/views/candidates/schools/_phase2.html.erb

```diff
@@ -207,11 +206,10 @@
 
     <% if include_candidate_request_links %>
       <div class="school-start-request-button__mobile">
-        <%= link_to "Start request",
+        <%= govuk_link_to "Start request",
                     new_candidates_school_registrations_personal_information_path(@presenter.school),
-                    class:"govuk-button",
                     aria: { label: "Start request for school experience at #{@presenter.school.name}" },
-                    data: { module: 'govuk-button' }%>
+                    data: { module: 'govuk-button' } %>
```

Comment:
Presumably the `data: { module` part can go now?

Reply:
Yep, well spotted!